robyn/argument_parser.py

```diff
@@ -66,7 +72,10 @@ def __init__(self) -> None:
 
         if self.dev and args.log_level is None:
             self.log_level = "DEBUG"
+        if args.version:
+            print(f'Python Version: {sys.version}')
+            sys.exit(0)    
```

Comment:
@TroyFPV48 👋 

Thank you for the PR 😄 Let us not use `sys.exit` here. We should add proper argument validation where we ensure that `--version` flag is called on its own.

Reply:
Sorry if i got this wrong. The intent of the issue was to print the robyn version instead of python version through the flag, right?

Instead of `sys.version` shouldn't it be this one? https://github.com/sparckles/robyn/blob/main/src/lib.rs#L26

Reply:
Thanks for catching this @banipreetr 😊

@TroyFPV48 , the intent of this issue is to print Robyn's version and not Python version.

Reply:
ahh i see thank you!

Reply:
@TroyFPV48 any updates on this?